backend/src/main/java/COMP_49X_our_search/backend/database/entities/Department.java

```diff
@@ -0,0 +1,78 @@
+package COMP_49X_our_search.backend.database.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "departments")
+public class Department {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Integer id;
+
+  @Column(nullable = false)
+  private String name;
+
+  @ManyToMany(mappedBy = "departments")
+  private Set<Faculty> faculties = new HashSet<>();
+
+  @ManyToMany(mappedBy = "departments")
+  private Set<Project> projects = new HashSet<>();
+
+  @OneToMany(mappedBy = "department")
```

Comment:
I think this is a typo and you meant "departments"

Reply:
I think the value of `mappedBy` has to correspond to the variable name that represents the relationship, and in `Major.java` it's currently `private Department department;` But it might have to change to "departments" if we confirm that there should be a Many-to-many relationship instead.

Reply:
Clarified with Dr. Clark, it should be Many-to-Many